pam/gdm/conversation.go

```diff
@@ -26,3 +27,137 @@ func sendToGdm(pamMTx pam.ModuleTransaction, data []byte) ([]byte, error) {
 	defer binRes.Release()
 	return binRes.Decode(decodeJSONProtoMessage)
 }
+
+// Send sends the data to the PAM Module, returning the JSON data.
+func (d *Data) Send(pamMTx pam.ModuleTransaction) ([]byte, error) {
+	bytes, err := d.JSON()
+	if err != nil {
+		return nil, err
+	}
+
+	return sendToGdm(pamMTx, bytes)
+}
+
```

Comment:
Mh, it looked convenient :)

Indeed the definition is elsewhere, but that's not something we can't avoid because of generation code. But I see also that it's not an operation related to data itself.

Reply:
Changed.